pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -663,16 +742,32 @@ def D_n(self, c_s_n, T):
         "Dimensionless negative particle diffusivity"
         sto = c_s_n
         T_dim = self.Delta_T * T + self.T_ref
-        return self.D_n_dimensional(sto, T_dim) / self.D_n_dimensional(
-            pybamm.Scalar(1), self.T_ref
+        try:
```

Comment:
probably cleaner to use an `if mechanics` type statement here to set `mech_effects` since you know in advance whether it will be 1 or not depending on the options

Reply:
corrected this